Foundation/NSPathUtilities.swift

```diff
@@ -10,13 +10,19 @@
 import CoreFoundation
 
 public func NSTemporaryDirectory() -> String {
-    #if os(macOS) || os(iOS)
+#if os(macOS) || os(iOS)
     var buf = [Int8](repeating: 0, count: 100)
```

Comment:
I really feel like this buffer should be dynamically allocated, as with the Windows version. `confstr()` does have the “returns the needed size” functionality.

Reply:
I think that's beyond the scope of this change, but I can do that in a different change.